lib/faulty/storage/redis.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'redis/cluster'
```

Comment:
We shouldn't load this since Redis isn't required for Faulty to load. Use `defined?` instead